### PR TITLE
Added new color variables

### PR DESCRIPTION
**Pull request by myueksel**
Added new color codes and color variables.

New colors:
colors v4

### CODE REVIEW

File: utils/variables/_colors.scss [hunk 1]
```diff
@@ -1,40 +1,130 @@
...
+$selago: #EAF0FD;
+$spindle: #BACAEB;
+$polo-blue: #7687AB;
+$lynch: #5D6B88;
+$fiord: #4A556C;
 
+// warm-blue - new 13-6-17
+$titan-white: #E5E6FF
+$perfume: #B2B5F8
+$burgundy-glow: #5F66DE
```
Comment (fragsalat) [inline utils/variables/_colors.scss]: Just recognized it, there are semicolons missing

File: utils/_schemas.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (faboweb) [inline utils/_schemas.scss]: nice!

File: utils/variables/_shadows.scss [hunk 3]
```diff
@@ -15,4 +15,15 @@ $elevation-shadows: (
     "raised": $elevation2,
     "hovering": $elevation3,
     "floating": $elevation5
-);
\ No newline at end of file
+);
+
+$paper-shadow-1: 0 1px 3px rgba($black, 0.12),
```
Comment (faboweb) [inline utils/variables/_shadows.scss]: why do we have two times the shadows now?

Comment (fragsalat) [inline utils/variables/_shadows.scss]: I saw that the shadows which are defined in invision differ with it's name and value from the shadows here. I don't wanted to make a big bang approach and remove the old shadows and therefore I first created the correct variables and in a second step later we will remove the old ones.

Comment (faboweb) [inline utils/variables/_shadows.scss]: what was the outcome for this?

Comment (fragsalat) [inline utils/variables/_shadows.scss]: I didn't saw Mehment today^^

Comment (faboweb) [inline utils/variables/_shadows.scss]: bump

**Comment by myueksel** [comment]
there is currently no use for paper shadow 3 & 5.
We can remove them for my opinion.

shadows and elevation v2

**Comment by fragsalat** [comment]
Ok so I'll remove them

**Comment by myueksel** [comment]
Yeah plz rename them... could you set the curren No.4 as first please.... thank you

**Comment by fragsalat** [comment]
So the new order would be 4, 1, 2, 6?
Done

**Pull request merged**